doc/specs/#754 - Cascading Default Settings.md

```diff
@@ -342,7 +342,7 @@ class IDynamicProfileGenerator
 ```
 
 For example, the WSL generator would return `Microsoft.Terminal.WSL`. The
-Powershell Core generator would return `Microsoft.Terminal.PowershellCore`.
+PowerShell Core generator would return `Microsoft.Terminal.PowershellCore`.

```

Comment:
```suggestion
PowerShell Core generator would return `Microsoft.Terminal.PowerShellCore`.

```

Reply:
You probably don't want that extra blank line. GitHub changed the default behavior of suggestion such that on average suggestions include one -- fairly annoying.